src/pages/settings/AddSecondaryLoginPage.js

```diff
@@ -102,7 +104,7 @@ class AddSecondaryLoginPage extends Component {
 
     render() {
         return (
-            <ScreenWrapper>
+            <ScreenWrapper onTransitionEnd={() => this.phoneNumberInputRef.focus()}>
```

Comment:
Add a falsey check on this, just to ensure that an uncaught exception isn't even possible?